docs/src/Guides/98 Migration from 4.X.md

```diff
@@ -66,20 +70,25 @@ modal = interactions.Modal(
 
 Otherwise, beyond renamings, components are largely the same.
 
-# WIP - these next sections are not in order of their final appearance
-
-## Extensions
+## Extensions (cogs)
 
 Extensions have not been changed too much. `await teardown(...)` is now just `drop(...)` (note how drop is *not* async), and you use `bot.load_extension`/`bot.unload_extension` instead of `bot.load`/`bot.unload`.
 
-There is one major difference though isn't fully related to extensions themselves - *you use the same decorator for both commands/events in your main file and commands/events in extensions in v5.* Basically, instead of having `bot.command` and `interactions.extension_command`, you *just* have `interactions.slash_command` (and so on for context menus, events, etc.), which functions seemlessly in both contexts.
+There is one major difference though that isn't fully related to extensions themselves: *you use the same decorator for both commands/events in your main file and commands/events in extensions in v5.* Basically, instead of having `bot.command` and `interactions.extension_command`, you *just* have `interactions.slash_command` (and so on for context menus, events, etc.), which functions seemlessly in both contexts.
 
 Also, you no longer require a `setup` function. They can still be used, but if you have no need for them other than just loading the extension, you can get rid of them if you want.
 
 ## Cache and interactions.get
 
-interactions.get is a mistake
-todo because im running out of battery—
+Instead of the `await interactions.get` function in v4, v5 introduces the `await bot.fetch_X` and `bot.get_X` functions, where `X` will be the type of object that you would like to retrieve (user, guild, role...). You might ask, what is the difference between fetch and get?
+
+The answer is simple, `get` will only look for an object that has been cached, and therefore is a synchronous function that can return None if this object has never been cached before.
+
+On the other hand, `fetch` is an asynchronous function that will request the Discord API to find that object if it has not been cached before. This ensures that you will always retrieve the most up to date version of that object, provided that the IDs you inputted are valid.
```

Comment:
```suggestion
On the other hand, `fetch` is an asynchronous function that will request the Discord API to find that object if it has not been cached before. This will *fetch* the latest version of the object from Discord, provided that the IDs you inputted are valid.
```